docs/how/search.md

```diff
@@ -179,6 +179,58 @@ for integrations and programmatic use-cases.
 ```
 
 
+For queries that return more than 10k entities we recommend using the [scrollAcrossEntities](https://datahubproject.io/docs/graphql/queries/#scrollacrossentities): 
```

Comment:
Can we add to a section called "Searching at Scale" 

?

Reply:
Sure